composer/utils/file_helpers.py

```diff
@@ -349,9 +349,15 @@ def maybe_create_object_store_from_uri(uri: str) -> Optional[ObjectStore]:
         return GCSObjectStore(bucket=bucket_name)
     elif backend == 'oci':
         return OCIObjectStore(bucket=bucket_name)
+    elif backend == 'dbfs':
+        if path.startswith('Volumes/'):
```

Comment:
I think the helpers here should also validate the 4 dirs thing? maybe `UCObjectStore` should expose a static method that just validates a path is the right format instead of duplicating here